token.go

```diff
@@ -78,53 +77,120 @@ func (c *Connection) TokensContext(ctx context.Context) (access, refresh string,
 		c.debugExpiry(ctx, "Bearer", c.accessToken, accessExpires, accessLeft)
 		c.debugExpiry(ctx, "Refresh", c.refreshToken, refreshExpires, refreshLeft)
 	}
-	if c.accessToken == nil || (accessExpires && accessLeft < 1*time.Minute) {
-		if c.refreshToken == nil || (refreshExpires && refreshLeft < 1*time.Minute) {
-			c.logger.Debug(ctx, "Requesting new token")
```

Comment:
did you remove this debug log on purpose or by accident?

Reply:
I moved it to the `sendRequestTokenForm` method.